practice-two/src/index.html

```diff
@@ -473,7 +473,42 @@ <h3 class="pricing__card-title">Enterprise</h3>
     <!-- END PRCING SECTION-->
 
       <!-- START CONTACT SECTION -->
-      <section></section>
+      <section class="contact">
+        <div class="container contact__wrapper">
+          <h2 class="heading-section text-color-base">Get in Touch</h2>
+          <form class="contact__form" action="javascript:void(0)">
+            <input type="text" name="name" placeholder="Name*" class="form-input contact__input">
+            <input type="email" name="email" placeholder="Email*" class="form-input contact__input">
+            <textarea name="message" placeholder="Message" class="form-input contact__input contact__message"></textarea>
+            <label class="text-normal text-color-base contact__form-center contact__label">
+              <input type="checkbox" class="contact__checkbox">
+              Most Urgent Question
+            </label>
+            <button type="submit" class="btn btn--contact text-color-secondary contact__form-center">Submit</button>
+          </form>
+          <address class="text-color-base contact__info">
+            <div class="contact__address">
+              <svg width="11" height="16">
+                <use xlink:href="assets/images/icon-contact.svg#location"></use>
+              </svg>
+              <p class="contact__text">2300 W Arbutus Dr Citrus Springs, FL 34434, USA</p>
+            </div>
+            <div class="contact__address">
+              <svg width="17" height="10">
+                <use xlink:href="assets/images/icon-contact.svg#email"></use>
+              </svg>
+              <a href="mailto:contact@hofmannui.net" class="contact__text contact__link">contact@hofmannui.net</a>
```

Comment:
You can use the BEM modifier here contact__text--hover instead of using an element contact__link here 
Class name contact_text is not good, you can use contact__detail or contact_item